jigglypuffRL/common/utils.py

```diff
@@ -18,13 +18,14 @@ def get_model(type_, name_):
     raise ValueError
 
 
-def mlp(sizes):
+def mlp(sizes, sac=False):
```

Comment:
This seems like a hack, no?

Reply:
Yeah it is! Was thinking of adding support for stochastic policies, so will change this once that's done. Is that okay?

Reply:
Ok sure